code/datums/components/plumbing/_plumbing.dm

```diff
@@ -22,10 +22,10 @@
 	var/recipient_reagents_holder
 	///How do we apply the new reagents to the receiver? Generally doesn't matter, but some stuff, like people, does care if its injected or whatevs
 	var/methods
-	///What color is our demand connect? Also it's not auto-colored so you'll have to make new sprites if its anything other than red, blue, yellow or green
-	var/demand_color = "red"
-	///What color is our supply connect? Also, refrain from pointlessly using non-standard colors unless it's really funny or something
-	var/supply_color = "blue"
+	///What color is our demand connect?
+	var/demand_color = "#ff0000"
```

Comment:
while it would be obvious what it is to hex color code readers, let's put that it's red in a comment beside this, and blue for the other

Reply:
Added for both colors